test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -4,11 +4,15 @@
 
 import org.junit.Test;
 
+import seedu.addressbook.data.exception.IllegalValueException;
+
 public class NameTest {
 
     @Test
-    public void test() {
```

Comment:
You should have commit an implemented function that does nothing to pass the first test case you created : "Not yet implemented"